Microsoft.Toolkit.Mvvm.SourceGenerators/AnalyzerReleases.Unshipped.md

```diff
@@ -0,0 +1,19 @@
+﻿; Unshipped analyzer release
+; https://github.com/dotnet/roslyn-analyzers/blob/master/src/Microsoft.CodeAnalysis.Analyzers/ReleaseTrackingAnalyzers.Help.md
+
+### New Rules
+
+Rule ID | Category | Severity | Notes
+--------|----------|----------|-------
+MVVMTK0001 | Microsoft.Toolkit.Mvvm.SourceGenerators.INotifyPropertyChangedGenerator | Error | See https://aka.ms/mvvmtoolkit
```

Comment:
When are these used? Are they actually errors? Should we create a specific aka.ms link for these specific warning/error codes and have that point to a doc which covers each one?

Reply:
These are reported by the source generators if anything invalid is detected, so it's a way to make the error reporting proactive and to result in build time errors instead of some weird runtime crash or some hard to understand error due to incorrect generated code resulting from wrong usage of some attribute or something. They're used eg. here:

https://github.com/CommunityToolkit/WindowsCommunityToolkit/blob/1b08e47a2314791356a6c8ed7fd19f7684b828af/Microsoft.Toolkit.Mvvm.SourceGenerators/ComponentModel/TransitiveMembersGenerator.cs#L58-L63

And yeah they're actual errors that will cause the build to fail until they're resolved 😄

Not sure whether we need a dedicated page given that they should already include enough info in their error message, but I suppose that's something we could put together if you think it'd be useful. Maybe just a single page with a paragraph for each error and a code snippet to show an example or something, and steps on how to resolve it. Thoughts?

Reply:
Yeah, I think that'd be good (they can just all be on a single page). I created a https://aka.ms/mvvmtoolkit/error link for us to point to that page (for now just points to doc root).

Reply:
@Sergio0694 want to update the links to the new aka.ms for future proofing? We can redirect it later wherever we need for this specific purpose without needing to make a code-update.

Reply:
Sure thing! Done in 806b6da6f5606fe5af6a3ab429b777db4331c59b 🙂